app-frontend/src/assets/styles/sass/layout/_container.scss

```diff
@@ -21,11 +21,11 @@
 
     &.dashboard {
       width: 100%;
-	    max-width: 1160px;
+	    // max-width: 1160px;
```

Comment:
For the styles, if I touch the main style sheets I tend to comment out rather than remove so that the design team can more easily ascertain the changes we've made.

This reminds me that I should've made a note of this: @designmatty, because we wanted a full width map preview at the top of the project detail page, we had to move the `max-width` rule to its own class and apply it as necessary within pages. This allows us to have a `div` (which has the map) with no `max-width` followed by a layout container where max-width is applied. There are more elegant ways to accomplish this, but it seems our usage of various high level container classes is a bit all over the place right now.